lib.sh

```diff
@@ -353,7 +353,10 @@ rh_finalize() {
     # Packaging cache is preserved across builds of container images
     $SUDO rm -f /etc/udev/rules.d/*-persistent-*.rules
     $SUDO touch /.unconfigured  # force firstboot to run
-    common_finalize
+
+    echo
+    echo "# PACKAGE LIST"
+    rpm -qa | sort
```

Comment:
Since we can't export artifacts, this is the next best thing. It is working as desired in my get-versions script.

Reply:
My script
1. reads the bot email
2. parses the table and links
3. fetches each log
4. reads each log, looking for `rpm -qa` and `dpkg -l` lines.